### PR TITLE
Document sig-multicluster decision about reserved namespaces

### PR DESCRIPTION
**Pull request by perotinus**
/assign @csbell @quinton-hoole 
/sig multicluster
/cc @pmorie @madhusudancs

Addresses https://github.com/kubernetes/cluster-registry/issues/221

### CODE REVIEW

File: contributors/design-proposals/multicluster/multicluster-reserved-namespaces.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    contains components, such as multicluster controllers and their
+    dependencies, that are not meant to be seen by most users directly.
+
+The definition of these namespaces is not intended to be exhaustive: in the
+future, there may be reason to define more multicluster namespaces, and
+potentially conventions for namespaces that are replicated between clusters (for
+example, to support a global cluster list that is replicated to all clusters
+that are contained in the list).
+
+## Conventions for reserved namesp[aces
```
Comment (csbell) [inline contributors/design-proposals/multicluster/multicluster-reserved-namespaces.md]: namesp[aces typo.

File: contributors/design-proposals/multicluster/multicluster-reserved-namespaces.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+By convention, resources in these namespaces are local to the clusters in which
```
Comment (csbell) [inline contributors/design-proposals/multicluster/multicluster-reserved-namespaces.md]: local or private? For multicluster, I reason about it at the logical level as private vs shared. Replication is a strategy that may or may not be applied to multicluster.

File: contributors/design-proposals/multicluster/multicluster-reserved-namespaces.md (hunk 1 above)
Comment (perotinus) [reply contributors/design-proposals/multicluster/multicluster-reserved-namespaces.md]: Thanks! Fixed.

**Pull request merged**